README.md

```diff
@@ -8,6 +8,35 @@ The step-by-step instructions [here](https://github.com/LEARNAcademy/Syllabus/bl
+
+**Incomplete**  
+An assessment that is considered incomplete is one that did not meet the requirements of a valid attempt on every question. Incomplete assessments will not be reviewed and will be returned to the student for completion. Students with incomplete assessments will not be able access the next assessment until the current one is complete.
+
+**Complete**  
+An assessment that is considered complete is one that met the bare minimum requirements of a valid attempt on every question. An assessment that is considered complete did not display enough effort to reach the novice level of skill acquisition.
+
+**Novice**  
+The novice stage is the first level of skill acquisition. In the novice level, the student is just beginning to understand the skill and requires a step-by-step "recipe" to to produce an output. An assessment that is considered at the novice level of skill acquisition is one that met the requirements of a valid attempt on every question while closely following process notes from class to produce:
+- basic pseudo code
+- code that will output content in the terminal
+- "your answers" written in the student's own words
+- researched answers written in the student's own words that provides additional context
+
+**Advanced Beginner**  
+The advanced beginner stage is the second level of skill acquisition. In the advanced beginner level, the student can begin to troubleshoot problems on their own and can decide if the "recipe" will be able to produce a particular outcome. An assessment that is considered at the advanced beginner level of skill acquisition is one that met the requirements of a valid attempt on every question while adapting process notes from class to produce:
+- pseudo code with proper vocabulary terms
+- code that will output correct content in the terminal
+- partial "your answers" written in complete sentences the student's own words
+- researched answers written in complete sentences in the student's own words that provides additional context
+
+**Competent**  
+The competent stage is the third level of skill acquisition. In the competent level, the student can decide what information is relevant to solve the problem and can being to create their own "recipe" to produce a particular outcome. An assessment that is considered at the competent level of skill acquisition is one that met the requirements of a valid attempt on every question while adapting process notes from class as well as from other resources to produce:
+- detailed pseudo code with proper vocabulary terms
+- code that will output correct content in the terminal following best practices
+- correct "your answers" written in complete sentences the student's own words that
```

Comment:
Line 41 Suggestion for clarity `correct "your answers" with the student's own words written in complete sentences`